kobuki_node/src/kobuki_ros.cpp

```diff
@@ -948,6 +946,12 @@ void KobukiRos::publishRawDataCommand(kobuki::Command::Buffer &buffer)
  */
 void KobukiRos::publishRawDataStream(kobuki::PacketFinder::BufferType &buffer)
 {
+  if (raw_data_stream_publisher_->get_subscription_count() == 0 &&
+    raw_data_stream_publisher_->get_intra_process_subscription_count() == 0)   // no one listening?
```

Comment:
This api is a little suprising. I would have inferred that `get_subscription_count` was the sum total of inter and intra subscriptions.

Reply:
Yeah, its a crufty part of the rclcpp API that I keep meaning to change, but have never gotten around to.